api/internal/handler/tool/tool.go

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package tool
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/shiningrush/droplet"
+	wgin "github.com/shiningrush/droplet/wrapper/gin"
+
+	"github.com/apisix/manager-api/internal/handler"
+	"github.com/apisix/manager-api/internal/utils"
+)
+
+type Handler struct {
+}
+
+type InfoOutput struct {
+	Hash    string `json:"commit_hash"`
+	Version string `json:"version"`
+}
+
+func NewHandler() (handler.RouteRegister, error) {
+	return &Handler{}, nil
+}
+
+func (h *Handler) ApplyRoute(r *gin.Engine) {
+	r.GET("/version", wgin.Wraps(h.Version))
```

Comment:
@starsz @nic-chen 

This will lead to 2 kinds of APIs:
1. `/apisix/admin/xxx`
2. `/version`

When we deprecate `admin-api` in V3, we will have to omit that prefix `apisix/admin` either, to have ManagerAPIs.

Now FE meets one issue: because HTTP call is handled by UmiJS, which only supports one prefix, `/apisix/admin` or `/`, then FE has to make hack codes to be compatible with those 2 kinds of APIs.

Could we add a prefix for this API? with some description about this case.

Reply:
OK.Got it.
I will add the prefix and ignore this API's auth checking in the next PR.

Reply:
I'm not sure if we need to omit the prefix, but unification is appropriate.

Reply:
Thanks!!!